frontend/appflowy_flutter/lib/shared/appflowy_network_image.dart

```diff
@@ -56,7 +57,12 @@ class FlowyNetworkImage extends StatelessWidget {
     final header = <String, String>{};
     final token = userProfilePB?.token;
     if (token != null) {
-      header['Authorization'] = 'Bearer ${jsonDecode(token)['access_token']}';
+      try {
+        final decodedToken = jsonDecode(token);
+        header['Authorization'] = 'Bearer ${decodedToken['access_token']}';
+      } catch (e) {
+        Log.error('unable to decode token: $e');
+      }
```

Comment:
Let's catch the specific exception
```suggestion
      } on FormatException catch (e) {
        Log.error('unable to decode token: $e');
      }
```